test/remote/recovery_email_resend_code_tests.js

```diff
@@ -7,6 +7,7 @@
 const assert = require('insist')
 const Client = require('../client')()
 var TestServer = require('../test_server')
+const P = require('bluebird')
```

Comment:
I'm totally cargo-culting this, but wouldn't we normally prefer `../../lib/promise` to requiring bluebird directly here?